mongorestore/testdata/tools-1856/js/d4.js

```diff
@@ -0,0 +1,15 @@
+// Dataset 7: Many (1,000) arrays
```

Comment:
```suggestion
// Dataset 4: Many (1,000) arrays
```
[nit] the comment doesn't match the filename

Reply:
Yeah, these were an unfortunate accident -- thanks for noticing this!